### DISCUSSION TITLE
Filecoin矿池资产管理工具

### DISCUSSION THREAD

**Discussion by Naturefir** [filecoin-project/community-china#31] [应用开发]
# [Filwallet](filwallet.ai)为大家提供了Filecoin矿池资产管理工具。
Filecoin矿池资产管理不仅操作繁琐，有安全风险，同时管理也有很大的门槛。很多矿工都无法自行管理集群资产。Filwallet为广大矿工提供了Filecoin矿池资产管理工具，可以通过手机APP发送命令进行资产管理。同时Filwalklet支持离线签名功能，可以在私钥和网络隔绝的状态下进行转账。

# Filwallet新功能
- 只读钱包：查看**任意矿工地址**和**矿工集群资产**情况
- 矿工钱包：查看矿池集群资产情况、各个地址**余额监控**
- 离线钱包：私钥和网络隔绝，通过离线签名的方式转账
- 离线签名转账
- Owner地址更换

# Filwallet只读钱包和矿工钱包页面
![只读钱包 矿工钱包-1](https://user-images.githubusercontent.com/79625747/111277716-b6baf980-8673-11eb-90ea-6af3c9c2703d.gif)

# Filwallet离线钱包页面
![离线钱包页面-1](https://user-images.githubusercontent.com/79625747/111278959-2087d300-8675-11eb-9c60-c2aa17420d4e.png)

# 下载方式
![下载二维码](https://user-images.githubusercontent.com/79625747/111278945-1cf44c00-8675-11eb-93e4-75758758f0f0.png)

# 使用反馈
欢迎大家尝鲜使用，大家在使用的过程中有任何疑问或者建议，欢迎提供给我们。您的每一条建议，都是我们不断完善产品的动力。可以添加VX：jyy321143521，回复“Filwallet”进Filewallet交流群。
Filwallet官网：https://filwallet.ai/
反馈表单：http://ipfsforce.mikecrm.com/DGu4W78

**Comment by Naturefir** [comment]
教程系列：
Filwallet教程-入门篇：https://www.bilibili.com/video/BV1hU4y1x7ow
Filwallet教程-矿工钱包功能说明：https://www.bilibili.com/video/BV1a54y1W7Tz
Filwallet教程-离线签名转账：https://www.bilibili.com/video/BV1KT4y1N7rN

**Reply by Naturefir** [reply]
欢迎来尝鲜使用哦~

**Reply by Fatman13** [reply]
包包🌹🌹🌹

**Reply by Naturefir** [reply]
🌹🌹🌹
